cinder/volume/drivers/lightos/lightos.py

```diff
@@ -619,18 +619,22 @@ def _wait_for_volume_available(
         # we can stop on any terminal status
         # possible states: Unknown, Creating, Available, Deleting, Deleted,
         # Failed, Updating
-        states_to_wait_for = (
-            'Available',
-            'Deleting',
-            'Deleted',
-            'Failed',
-            'UNKNOWN')
-        return self._wait_for_volume_state(
-            project_name,
-            timeout=timeout,
-            states=states_to_wait_for,
-            vol_uuid=vol_uuid,
-            vol_name=vol_name)
+        states = ('Available', 'Deleting', 'Deleted', 'Failed', 'UNKNOWN')
```

Comment:
note that lightOS will never return Deleted

not for now, but need to discuss what we want to do for states rollback and in particular migrating